README.md

```diff
@@ -4,20 +4,37 @@ This repository contains a performance test setup for a Nuts network.
 It starts multiple nuts nodes. 
 For each node a prometheus datasource is used to display graphs in grafana.
 
+
+## Networking
+
+Monitoring is started separately from the performance tests.
+This way the monitoring setup can always stay running in the background while new tests are started.
+For this to work, it's easiest to create a shared network so prometheus has access to all relevant endpoints for scraping.
```

Comment:
better to call it a hard requirement. Which is done in all the docker compose files in this repo